<!DOCTYPE html>
<html lang="en">

<head>
    <title>Research</title>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="CSS\style_1.css">
    <style>
        body {
            font-family: "Lato", sans-serif
        }

        .mySlides {
            display: none
        }
    </style>
</head>

<body>

    <!-- Navbar -->
    <div class="w3-top">
        <div class="w3-bar w3-indigo w3-card">
            <a class="w3-bar-item w3-button w3-padding-large w3-hide-medium w3-hide-large w3-right"
                href="javascript:void(0)" onclick="myFunction()" title="Toggle Navigation Menu"><i
                    class="fa fa-bars"></i></a>
            <a href="index.html" class="w3-bar-item w3-button w3-padding-large">Home</a>
            <a href="Research.html" class="w3-bar-item w3-button w3-padding-large w3-hide-small">Research</a>
            <a href="publication.html" class="w3-bar-item w3-button w3-padding-large w3-hide-small">Publications</a>
            <a href="Teaching.html" class="w3-bar-item w3-button w3-padding-large w3-hide-small">Teaching</a>
            <a href="News.html" class="w3-bar-item w3-button w3-padding-large w3-hide-small">News</a>
            <a href="Team.html" class="w3-bar-item w3-button w3-padding-large w3-hide-small">Team</a>
            <a href="gallery.html" class="w3-bar-item w3-button w3-padding-large w3-hide-small">Gallery</a>
            <div class="w3-dropdown-hover w3-hide-small">
                <button class="w3-padding-large w3-button" title="More">Tools</button>
                <div class="w3-dropdown-content w3-bar-block w3-card-4">
                    <a href="Equipment.html" class="w3-bar-item w3-button">Equipment</a>
                    <a href="HPC.html" class="w3-bar-item w3-button">Software</a>
                </div>
            </div>
            <a href="javascript:void(0)" class="w3-padding-large w3-hover-red w3-hide-small w3-right"><i
                    class="fa fa-search"></i></a>
        </div>
    </div>

    <!-- Navbar on small screens (remove the onclick attribute if you want the navbar to always show on top of the content when clicking on the links) -->
    <div id="navDemo" class="w3-bar-block w3-indigo w3-hide w3-hide-large w3-hide-medium w3-top"
        style="margin-top:46px">
        <a href="publication.html" class="w3-bar-item w3-button w3-padding-large"
            onclick="myFunction()">Publications</a>
        <a href="Research.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">Research</a>
        <!-- <a href="News.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">News</a> -->
        <a href="Teaching.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">Teaching</a>
        <a href="Team.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">Team</a>
        <a href="gallery.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">Gallery</a>
        <a href="Equipment.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">Equipment</a>
        <a href="HPC.html" class="w3-bar-item w3-button w3-padding-large" onclick="myFunction()">Software</a>
    </div>
    <div class="container1" style="background-color: rgb(247, 247, 247);">
        <div class="image1">
            <img src="image\logo iitkgp.jpeg" style="height: 65px; height: 75px;">
        </div>
        <div class="text1">
            <h1>SYSTEMS AND SYNTHETIC BIOLOGY LABORATORY</h1>
        </div>
    </div>
    <div id="contener">
        <div class="box1">
            <h3>Our research domains</h3>
        </div>

        <div class="w3-row w3-margin w3-animate-left">

            <div class="w3-third">
                <font size="5" face="justify"><b>CRISPR/Cas9 for Genome Engineering :</b></font> <br>
                <img src="image/slider_2.jpg" style="width:100%;min-height:300px ">
            </div>
            <div class="w3-twothird w3-container">
                <font size="4" face="justufy"><br>

                    Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) and CRISPR-associated (Cas)
                    proteins were
                    discovered in prokaryotes as an adaptive immune system against viral and plasmidal genome invasion.
                    The type
                    II CRISPR/Cas system uses a tracrRNA:crRNA hybrid to bind to Cas9 protein and guide it to create a
                    site-specific double stranded break on target templates. Modification of the tracrRNA:crRNA hybrid
                    to a single
                    guide RNA that has a customizable recognition sequence to guide DNA cleavage has become a very
                    popular
                    technique in gene editing. CRISPR/Cas9 technique also allows a marker-less genome manipulation that
                    enables
                    several modifications on the same strain. Post-cleavage of DNA with CRISPR/Cas9, the DNA repair
                    occurs via
                    non-homologous end joining (NHEJ) or homologous recombination (HR).<br>An increasing global energy
                    demand
                    and environmental concerns associated with petroleum fuels have necessitated the need for
                    alternative liquid
                    fuels (biofuels). Microbes such as yeasts can utilize sugars (generally hexoses) to produce
                    different
                    compounds such as free fatty acids (FFA) and its derivatives such as fatty-acid ethyl ester (FAEE),
                    fatty
                    alcohols and alkane/alkene. These biomolecules have energy content comparable to diesel and petrol,
                    making
                    them an important candidate for biofuels. However, yeasts do not generally utilize hexose sugars
                    such as
                    xylose due to lack of intrinsic pathways and transporters. In our lab, we are utilizing CRISPR/Cas9
                    tool to
                    engineer the metabolic pathways of different yeast strains for co-utilization of glucose and xylose
                    and
                    produce fatty acids (biofuels). We are also developing the CRISPR/Cas9 tool for certain industrially
                    relevant
                    yeasts.
                </font>
            </div>

        </div>
        <div class="w3-row w3-margin w3-animate-bottom">

            <div class="w3-third">
                <font size="5" face="justify"><b>Metabolic Flux Analysis and Metabolomics:</b></font> <br>
                <img src="image/F1.jpg" style="width:100%;min-height:300px">
            </div>
            <div class="w3-twothird w3-container">
                <font size="4" face="justufy"><br>
                    Metabolic Flux Analysis (MFA) is an important method for the quantitative estimation of
                    intracellular
                    metabolic flows through metabolic pathways in a variety of cellular metabolism of relevance to
                    medicine and
                    biotechnology. For more complete understanding of biochemical pathways along with a quantitative
                    analysis,
                    metabolic flux analysis is an important technique. Metabolic flux analysis is generally carried out
                    with
                    isotopic tracers (e.g., 13C) to elucidate the pathways followed by labeled metabolites. Mass
                    spectrometry is
                    absolutely instrumental to record the labeling patterns in metabolic intermediates and end products.
                    Our
                    current efforts aim at developing new approaches to identify the rate limiting steps in sugar
                    utilization for
                    biofuels production in the engineered yeast strains.
                </font>
            </div>

        </div>
        <div class="w3-row w3-margin">

            <div class="w3-third">
                <font size="5" face="justify"><b>Molecular Dynamics Simulation :</b></font> <br>
                <img src="image/Cel.gif" style="width:100%;min-height:300px">
                <p><b> microcrystal solvated in an IL-Water solvent mixture</b></p>
            </div>
            <div class="w3-twothird w3-container">
                <font size="4" face="justufy"><br>
                    Molecular Dynamics Simulation is a classical mechanics based computational approach that enables to
                    investigate the behaviour of a microsystem consisting a large number of interacting particles and to
                    predict
                    the bulk properties of the macrosystem at molecular level. GPU accelerated high-performance
                    computing has
                    revolutionized the theoretical approached to study the structure and dynamics of biological
                    macromolecules
                    from a nanosecond (ns) to milliseconds (ms) timescale in recent years.<br>Our lab is particularly
                    interested to explore the molecular mechanism of various biological processes in the domain of
                    carbohydrates
                    as well as protein chemistry using MD simulation. Currently, we are focussing to understand the
                    dissolution of
                    cellulose in Ionic liquid (1-Ethyl-3-methylimidazolium acetate) and water mixtures for designing a
                    cost-effective and optimized lignocellulosic biomass pretreatment process using IL. We are also
                    working to
                    develop theoretical insights behind the IL tolerance of thermostable cellulolytic enzymes, which
                    will guide
                    the experimental biologists for engineering the existing cellulase with improved saccharification
                    kinetics.
                    Moreover, molecular simulations of proteins reveal multiple conformational states in a trajectory
                    ensemble
                    which can explain its in-vivo functionality. Steered Molecular Dynamics are also routinely carried
                    out in our
                    lab to study the force induced unfolding of proteins.
                </font>
            </div>

        </div>
        <div class="w3-row w3-margin">

            <div class="w3-third">
                <font size="5" face="justify"><b>Systems Biology of Gut Microbiome :</b></font> <br>
                <img src="for_web\1-s2.0-S2589014X22000469-gr1.jpg" style="width:100%;min-height:300px">
            </div>
            <div class="w3-twothird w3-container">
                <font size="4" face="justufy"><br>
                    Diverse environmental condition reinforce the growth of different microbial communities with
                    specialized characteristic. Metagenomic study along with computational technique disclose the
                    structural and functional complexity of microbial niche. Culture independent approach of
                    metagenomics helps to conquer the hurdle of characterization of unculturable microbes, enhancing
                    clarity of community based microbiology. Gut microbiota has been one of the most fascinating area of
                    research during several decades because of enormous microbial diversity and their metabolic
                    potential. We are interested to study the cumulative potential of different gut microbiota that
                    effectively translate the lignocellusoic biomass to biofuel. A system level understanding of the
                    microbial interdependency and cross communication will decrypt the natural system lignocellulose
                    degradative system leading towards the designing of a successful invitro mechanism for effective
                    biomass conversion for biofuel production. Systematic mathematical model development of inter
                    species interaction will help to identify key-species (influencer) in microbial communities having
                    powerful lignocellulolytic property.
                </font>
            </div>

        </div>


    </div>




    <script>
        // Used to toggle the menu on small screens when clicking on the menu button
        function myFunction() {
            var x = document.getElementById("navDemo");
            if (x.className.indexOf("w3-show") == -1) {
                x.className += " w3-show";
            } else {
                x.className = x.className.replace(" w3-show", "");
            }
        }
    </script>
</body>

</html>